unittests/test_iem.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) 2001-2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+import os
+import sys
+import unittest
+
+from cortx.utils.iem_framework import EventMessage
+from cortx.utils.process import SimpleProcess
+
+PROJECT_ROOT = "/".join(os.path.abspath(__file__).split("/")
+                        [:-2]) + "/low-level"
+sys.path.append(PROJECT_ROOT)
```

Comment:
We don't need this. This path is already imported in `unittests/conftest.py` which is config file for pytest.

Reply:
done